### PR TITLE
completed css-flexbox exercise

### PR DESCRIPTION
**Pull request by joseph-w-valdez**
WBAdHZu
align-items

### CODE REVIEW

**Comment by joseph-w-valdez** [comment]
JVbgKio
justify content

**Comment by joseph-w-valdez** [comment]
PQfeWHu
flex-wrap

**Comment by joseph-w-valdez** [comment]
jvuR4C3
display: flex

File: css-flexbox/display-flex.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    #card-deck {
+display:flex;
+flex-direction:row;
+padding:2rem;
+    }
+    .card-wrapper {
+padding:0.5rem;
+
+    }
+}
```
Comment (codingcodymiller) [inline css-flexbox/display-flex.html:20]: WOAH

Please fix this indentation 🤢

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-flexbox/display-flex.html (hunk 1 above)
Comment (joseph-w-valdez) [reply css-flexbox/display-flex.html:20]: oh jeeze. i get these messes often since i'm still not used to deleting lines while holding shift

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**